buildSrc/src/main/rocker/com/uber/okbuck/template/config/BuckDefs.rocker.raw

```diff
@@ -104,7 +105,10 @@ def okbuck_lint(
     bash += "-classpath {} com.android.tools.lint.Main ".format(toLocation('//.okbuck/workspace/lint:okbuck_lint'))
 
     if has_srcs:
-        bash += "--classpath $(location :src_{}) --libraries `cat $CP_FILE` ".format(variant)
+@if(useCompilationClasspath) {
+        bash += "--classpath $(location :src_{}) "
```

Comment:
`.format(variant)` should be here

Reply:
should we also add `bash += "CP_FILE=`sed 's/@@//' <<< $(@@@(classpathMacro) :src_{})`; ".format(variant)` conditionally using `useCompilationClasspath`, since classpath abi will require compilation as well

Reply:
classpath abi does not include abi of src_lib target does it?